Tests/SotoCoreTests/AWSRequestTests.swift

```diff
@@ -12,7 +12,7 @@
 //
 //===----------------------------------------------------------------------===//
 
-#if compiler(<5.7)
+#if compiler(<5.7) || os(Linux)
```

Comment:
Looks like you missed changing the `#if`s here and in `DictionaryEncoderTests.swift`, if this isn't on purpose

Reply:
Yep looks like I missed that one

Reply:
fixed